routes/blog/[slug].tsx

```diff
@@ -33,6 +34,15 @@ export default function PostPage(props: PageProps<Post>) {
       <Head>
         <style dangerouslySetInnerHTML={{ __html: CSS }} />
         <title>{post.title}</title>
+        <meta property="og:type" content="website" />
+        <meta property="og:site_name" content={SITE_NAME} />
+        <meta property="og:locale" content="en" />
+        <meta property="og:title" content={post.title} />
+        <meta property="og:description" content={post.summary} />
+        <meta property="og:url" content={props.url.origin} />
+        <meta name="twitter:title" content={post.title} />
+        <meta name="twitter:description" content={post.summary} />
+        <meta name="description" content={post.summary} />
```

Comment:
Are we able to add `<meta itemprop="name"...>` and `<meta itemprop="description"... >`?

Reply:
Of course, I added it to `/blog/[slug].tsx` and `/blog/index.tsx` respectively